library/tasks/HISAT2.wdl

```diff
@@ -211,6 +211,7 @@ task HISAT2SingleEnd {
   # Need room for unsorted + sorted bam + temp sorting space + zipped and unzipped ref. Add 10 GB buffer.
   Int disk = ceil((size(fastq, "GB") * 100) + size(hisat2_ref, "GB") * 2 + 10)
```

Comment:
Confirm with @kbergin if these should be GiBs or not yet

Reply:
According to [the spec](https://github.com/openwdl/wdl/blob/master/versions/1.0/SPEC.md#float-sizefile-string), either is supported for the `size` call, but it looks like [Cromwell expects](https://cromwell.readthedocs.io/en/stable/RuntimeAttributes/#disks) that number to mean GB.

Reply:
A single PR should update `GB` -> `GiB` if/when you decide to make the change; until then users should be expected to pattern match on existing code as Marcus has done.

Reply:
There was a change in Cromwell recently so that the size function returns GiB even if you ask for GB so we should definitely change this to use GiB to be explicit: https://github.com/broadinstitute/cromwell/issues/4906

Reply:
Okay, updated to `GiB`